contrib/examples/actions/workflows/orquesta-ask-multiple-parameters.yaml

```diff
@@ -0,0 +1,56 @@
+version: 1.0
+
+description: A basic workflow that demonstrates inquiry with multiple parameters.
+
+tasks:
+  start:
+    action: core.echo message="Automation started."
+    next:
+      - when: <% succeeded() %>
+        do: get_approval
+
+  get_approval:
+    action: core.ask
+    input:
+      schema:
+        type: object
+        properties:
+          approved:
+            type: boolean
+            description: "Continue?"
+            required: True
+          department_id:
+            type: number
+            description: "Your department ID:"
+            required: True
+          extra_output:
+            type: string
+            description: "Your message to echo next if you approve to continue:"
+            default: "None"
+      ttl: 60
+    next:
+      - when: <% task(get_approval).result.response.extra_output != 'None' and task(get_approval).result.response.approved = true %>
+        publish: 
+          - custom_output: <% task(get_approval).result.response.extra_output %>
+          - approver_department_id: <% task(get_approval).result.response.department_id %>
+        do: echo_extra_message
+      - when: <% task(get_approval).result.response.approved = true %>
```

Comment:
This condition will be triggered regardless if extra_output is None.  So in the case where extra_output is not None, both the `echo_extra_message` and `finish` tasks will be triggered simultaneously.  Is this your intention?  If not, then the condition here should be `<% task(get_approval).result.response.extra_output = 'None' and task(get_approval).result.response.approved = true %>`

Reply:
Ok, I may have a misunderstanding here. So, my assumption was that the different when conditions are evaluated in their order. As soon as one condition matches the evaluation of the remaining ones will be skipped. 

So my idea was: 
- if we have the approval AND a custom message -> echo_extra_message which will trigger the finishing action (the remaining two conditions won't be checked)
- if stackstorm proceeds to the 2nd condition we know that we got the approval but no extra message (otherwise the condition above would have matched)

Is this assumption wrong? I'll also do some more tests tomorrow to see what I missed during my recent tests.

Reply:
Yes, your assumption here is not accurate. The transition is evaluated in order but will be triggered if the condition is met. So in this example, multiple task transitions can be triggered.

Reply:
Ok, got it. Somehow, I get another error when I do not provide any extra_output: 
```      "message": "YaqlEvaluationException: Unable to resolve key 'extra_output' in expression '<% task(get_approval).result.response.extra_output != 'None' and task(get_approval).result.response.approved = true %>' from context.",```

This makes sense because the response sent by the browser contains just `"response":{"approved":true,"department_id":41411}`. So the default is not part of the response object. 

I also tried `extra_output in task(get_approval.result.response` but this seems not to be supported. I will check after lunch how to proceed here. Might be a YAQL expression that's needed here.

Reply:
Ok, was able to reproduce the issue and see the behaviour you described. PR will be updated shortly.